sites/playground/src/routes/tests/while-in-view.svelte

```
@@ -9,7 +9,59 @@ const once = $derived(page.url.searchParams.get('once') ? true : undefined);
+type StaggerUpdate = {
+	opacity: number;
+	time: number;
+	y: number;
+};
+
+type StaggerWindow = Window & {
+	__staggerUpdates?: Record<number, StaggerUpdate[]>;
+};
+
+function recordStaggerUpdate(number: number, latest: Record<string, string | number>) {
+	const updates = ((window as StaggerWindow).__staggerUpdates ??= {});
```

Biome (2.5.5)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)